helm/hpcc/templates/_helpers.tpl

```diff
@@ -2184,3 +2184,19 @@ Pass in dict with resources object
  {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Pass in dict with me
+Fills "result" dictionary with "planeCategories" and "namePlanes"
+*/}}
+{{- define "hpcc.getEnginePlanes" -}}
+{{- $planeCategories := list "lz" "data" "remote" "dll" "debug" -}}
+{{- $namedPlanes := list -}}
+{{- if not .me.spillPlane -}}
+ {{- $planeCategories = append $planeCategories "spill" -}}
```

Comment:
Better to pick the first spill plane?  I don' t think the others will ever be used anyway.

Reply:
agree, changed in new commit.